src/main/scala/org/broadinstitute/dsde/agora/server/model/AgoraEntity.scala

```diff
@@ -41,7 +41,9 @@ case class AgoraEntity(@(ApiModelProperty@field)(required = false, value = "The
                        url: Option[String] = None,
                        @(ApiModelProperty@field)(dataType = "string", required = false, value = "The type of the entity (Task, Workflow, Configuration)", allowableValues = "Task,Workflow,Configuration")
                        entityType: Option[AgoraEntityType.EntityType] = None
-                        )
+                        ) {
+  def getUniqueIdentifier: String = namespace + ":" + name + ":" + snapshotId
```

Comment:
Currently I'm only using this in testing frameworks so it might make sense to just push it down to a testing companion object.